models/classes/class.QtiTestService.php

```diff
@@ -632,7 +632,8 @@ protected function importTest(
                         }
                     }
 
-                    $this->deleteTestsFromClassByLabel($testLabel, $itemsClassLabel, $testClass, $itemParentClass);
+                    $this->getTestService()->deleteTest(new core_kernel_classes_Resource($overwriteTestUri));
```

Comment:
- [ ] @shpran We cannot forget to move the test to the new path before deleting it (so we do a clean up).
- [ ] @shpran We cannot forget to move the test items to the new path before deleting them (so we do a clean up).

Reply:
We provide new path in synchronizer app, so this part already done 🙂 
First - we build new path
Second - we publish the test
During publishing:
First - we delete old one
Second - we publish new one